sources/vcdMaker/src/vcdMaker.cpp

```diff
@@ -80,6 +80,10 @@ int main(int argc, const char *argv[])
         LOGGER::Logger::GetInstance().LogError(rException);
         executionStatus = EXECUTION::APP_ERROR;
     }
+    catch (TCLAP::CmdLineParseException &)
+    {
+        executionStatus = EXECUTION::APP_ERROR;
```

Comment:
Tclap is returning its own syntax error information so I decided that it would be over-done to log an additional error. It seems to me that the information provided by Tclap is clean and sufficient.